src/commcare_cloud/ansible/roles/formplayer/tasks/main.yml

```diff
@@ -120,7 +120,6 @@
     mode: 0644
   with_items:
     - filename: application.properties
-    - filename: sentry.properties
```

Comment:
Looks like the idea is that by just _not copying_ `sentry.properties` to the new release, we're essentially "deleting" it, and that's why there's no cleanup task necessary here. Took me a second, but makes sense

Reply:
Yes, that's correct.